Source/Examples/Avalonia/ExampleBrowser/Program.cs

```diff
@@ -7,8 +7,14 @@ class Program
         // Initialization code. Don't use any Avalonia, third-party APIs or any
         // SynchronizationContext-reliant code before AppMain is called: things aren't initialized
         // yet and stuff might break.
-        public static void Main(string[] args) => BuildAvaloniaApp()
+        public static void Main(string[] args)
+        {
+            OxyPlot.Avalonia.OxyPlotModule.EnsureLoaded();
+            OxyPlot.SkiaSharp.Avalonia.OxyPlotModule.EnsureLoaded();
```

Comment:
What purpose do these serve?

Reply:
No idea, I have this from some Avalonia example which I can't find anymore. This seemed to be required at some point when I was testing, but I just tried without these lines and it seems to work fine...

Reply:
I had wondered if it was to force the XML namespaces or something, but I don't know.